src/Sarif.ValidationTests/Sarif.ValidationTests.csproj

```diff
@@ -45,6 +51,5 @@
     <Copy SourceFiles="@(v2ConverterTestData)" DestinationFolder="$(OutputPath)v2\ConverterTestData\%(RecursiveDir)" />
     <Copy SourceFiles="@(v2DirectProducerTestData)" DestinationFolder="$(OutputPath)v2\DirectProducerTestData\%(RecursiveDir)" />
     <Copy SourceFiles="@(v2SpecExamples)" DestinationFolder="$(OutputPath)v2\SpecExamples\%(RecursiveDir)" />
-    <Copy SourceFiles="$(SolutionDir)\Sarif\Schemata\Sarif.schema.json" DestinationFolder="$(OutputPath)" />
```

Comment:
For some reason, performing a copy like this makes copies both in the actual output directory (e.g., bld\bin\AnyCPU_Release\Sarif.ValidationTests\net461) and unnecessarily in the parent directory (e.g., bld\bin\AnyCPU_Release\Sarif.ValidationTests). Using the <None> element with CopyToOutputDirectory metadata item does what I want.

Come to think of it, I should make this change for the other copied files as well. I see that there are extra unnecessary v1 and v2 subdirectories directly under bld\bin\AnyCPU_Release\Sarif.ValidationTests.